x-pack/plugins/timelines/public/index.ts

```diff
@@ -52,6 +52,7 @@ export {
   getTimelineIdFromColumnDroppableId,
 } from './components/drag_and_drop/helpers';
 export { StatefulFieldsBrowser } from './components/t_grid/toolbar/fields_browser';
+export { BulkActions } from './components/t_grid/toolbar/bulk_actions';
```

Comment:
I'm working on doing something like this for add to case but it's not quite done yet, but I think instead of just exporting this at the top level, which increases bundle size, we should do something like this at the plugin level, where we asynchronously load what is exported here:
```

      getAddToCaseHook: () => {
        return async function () {
          /**
           * The specially formatted comment in the `import` expression causes the corresponding webpack chunk to be named. This aids us in debugging chunk size issues.
           * See https://webpack.js.org/api/module-methods/#magic-comments
           */
          const addToCase = await import(
            /* webpackChunkName: "timelinesExports" */ './hooks/use_add_to_case'
          );
          return addToCase;
        };
      },
```
(adjust for bulk actions, this is the cases function I'm using) this will prevent having to bump the plugin bundle size I think

Reply:
Actually I see that bulk actions is just a component, should just use React.lazy in that case